newIDE/app/src/VersionHistory/UseVersionHistory.js

```diff
@@ -22,6 +22,24 @@ import { SubscriptionSuggestionContext } from '../Profile/Subscription/Subscript
 import useAlertDialog from '../UI/Alert/useAlertDialog';
 import PlaceholderLoader from '../UI/PlaceholderLoader';
 import type { MessageDescriptor } from '../Utils/i18n/MessageDescriptor.flow';
+import PlaceholderError from '../UI/PlaceholderError';
+
+const getCloudProjectFileMetadataIdentifier = (
+  storageProviderInternalName: string,
+  fileMetadata: ?FileMetadata
+) => {
+  if (!fileMetadata || !storageProviderInternalName === 'Cloud') return null;
+  if (fileMetadata.fileIdentifier.startsWith('http')) {
+    // When creating a cloud project from an example, there might be a moment where
+    // the used Storage Provider is the cloud one but the file identifier is the url
+    // to the example such as `https://ressources.gdevelop-app.com/...`.
+    // A cloud project identifier is a uuid at the moment, so it does not contain the
+    // 3 letters h t and p so there should be no risk af having a cloud project
+    // uuid starting with http.
+    return null;
```

Comment:
Have you observed a problem when loading an example? What was it?

Reply:
UseVersionHistory tried to load versions with the URL `https://api.gdevelop.io/project/project/https://resources.gdevelop-app.com/.../version`
And it was crashing the app.
So I added the error message + this safety